CHANGELOG.md

```diff
@@ -2,7 +2,42 @@
 All notable changes to this project will be documented in this file.
 See the [change log guidelines](http://keepachangelog.com/) for information on how to structure the file.
 
-## [Unreleased]
+## [5.1.0]
+
+### Added
+- Support for reading multiple sheets from an Excel file simultaneously
+- Enable exporting of multiple feature types to Excel
+- Allow loading Excel with blank rows and sheets for schema and source data
+- Add predefined and custom formatters for Date cells in Excel import
+- Enable exporting feature types with no data in Excel
+- Align Excel with CSV export wizard
+- Prefer properties with short paths as GeoJSON geometry property
+- Add wizard page to select the read mode for GeoJSON/JSON
+- Extend JSON reader functionality
+- Support loading schema from JSON data
+- Configure decimal places in JSON
```

Comment:
From the description it is hard to determine what the change is.
Is this for export and import of Json data? Is it supported in the UI?

Reply:
Any better:

- Added support for importing data from JSON files, enhancing the versatility of data import options.
- Introduced support for loading JSON schema, providing users with greater flexibility in handling JSON data structures.